lms/djangoapps/instructor_task/tasks_helper/misc.py

```diff
@@ -518,7 +518,7 @@ def _log_and_update_progress(step):
     task_progress.attempted = students.count
     _log_and_update_progress({'step': "Finished compiling learner rows"})
 
-    filename = str(course_id).replace('/', '-') + '-anon-ids'
-    upload_csv_to_report_store([header] + rows, filename, course_id, start_date)
+    csv_name = 'anonymized_ids'
+    upload_csv_to_report_store([header] + rows, csv_name, course_id, start_date)
```

Comment:
why we are changing it to `anonymized_ids` name?

Reply:
the method is expecting the `csv` file type and not the full `filename` (that is handled within method)... `anon-ids` => `anonymized_ids` was done for more clarity.